### PR TITLE
update js dependencies

### PR DESCRIPTION
**Pull request by nl0**

### CODE REVIEW

File: catalog/app/containers/Bucket/File.js [hunk 1]
```diff
@@ -191,7 +191,7 @@ function Analytics({ analyticsBucket, bucket, path }) {
   const formatDate = (date) =>
     dateFns.format(
       date,
-      today.getFullYear() === date.getFullYear() ? `D MMM` : `D MMM YYYY`,
+      today.getFullYear() === date.getFullYear() ? `d MMM` : `d MMM yyyy`,
```
Comment (akarve) [inline catalog/app/containers/Bucket/File.js]: do we really need a template string here?

Comment (nl0) [reply catalog/app/containers/Bucket/File.js]: nope. good catch. i wonder why prettier didnt catch that

Comment (nl0) [reply catalog/app/containers/Bucket/File.js]:
```suggestion
      today.getFullYear() === date.getFullYear() ? 'd MMM' : 'd MMM yyyy',
```

**Comment by mistercrunch** [comment]
# [Codecov](https://codecov.io/gh/quiltdata/quilt/pull/1523?src=pr&el=h1) Report
> Merging [#1523](https://codecov.io/gh/quiltdata/quilt/pull/1523?src=pr&el=desc) into [master](https://codecov.io/gh/quiltdata/quilt/commit/d180c5ab52b2c7702f5b07c85ff689174ecfaf64&el=desc) will **not change** coverage by `%`.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/quiltdata/quilt/pull/1523?src=pr&el=tree)

```diff
@@           Coverage Diff           @@
##           master    #1523   +/-   ##
=======================================
  Coverage   83.31%   83.31%           
=======================================
  Files          27       27           
  Lines        3715     3715           
=======================================
  Hits         3095     3095           
  Misses        620      620           
```



------

[Continue to review full report at Codecov](https://codecov.io/gh/quiltdata/quilt/pull/1523?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/quiltdata/quilt/pull/1523?src=pr&el=footer). Last update [d180c5a...d180c5a](https://codecov.io/gh/quiltdata/quilt/pull/1523?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Comment by mistercrunch** [comment]
# [Codecov](https://codecov.io/gh/quiltdata/quilt/pull/1523?src=pr&el=h1) Report
> Merging [#1523](https://codecov.io/gh/quiltdata/quilt/pull/1523?src=pr&el=desc) into [master](https://codecov.io/gh/quiltdata/quilt/commit/674d05b4a489b9c7c0d4f8fbabf8e1f533efecea&el=desc) will **decrease** coverage by `0.02%`.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/quiltdata/quilt/pull/1523?src=pr&el=tree)

```diff
@@            Coverage Diff             @@
##           master    #1523      +/-   ##
==========================================
- Coverage   83.33%   83.31%   -0.03%     
==========================================
  Files          27       27              
  Lines        3715     3715              
==========================================
- Hits         3096     3095       -1     
- Misses        619      620       +1     
```


| [Impacted Files](https://codecov.io/gh/quiltdata/quilt/pull/1523?src=pr&el=tree) | Coverage Δ | |
|---|---|---|
| [quilt3/search\_util.py](https://codecov.io/gh/quiltdata/quilt/pull/1523/diff?src=pr&el=tree#diff-cXVpbHQzL3NlYXJjaF91dGlsLnB5) | `75.86% <0.00%> (-3.45%)` | :arrow_down: |
| [quilt3/util.py](https://codecov.io/gh/quiltdata/quilt/pull/1523/diff?src=pr&el=tree#diff-cXVpbHQzL3V0aWwucHk=) | `78.94% <0.00%> (ø)` | :arrow_up: |

------

[Continue to review full report at Codecov](https://codecov.io/gh/quiltdata/quilt/pull/1523?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/quiltdata/quilt/pull/1523?src=pr&el=footer). Last update [5ab5538...5ab5538](https://codecov.io/gh/quiltdata/quilt/pull/1523?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Comment by mistercrunch** [comment]
# [Codecov](https://codecov.io/gh/quiltdata/quilt/pull/1523?src=pr&el=h1) Report
> Merging [#1523](https://codecov.io/gh/quiltdata/quilt/pull/1523?src=pr&el=desc) into [master](https://codecov.io/gh/quiltdata/quilt/commit/5ab5538f4c4bcde0f769e6b784776347e0c9abc7&el=desc) will **not change** coverage by `%`.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/quiltdata/quilt/pull/1523?src=pr&el=tree)

```diff
@@           Coverage Diff           @@
##           master    #1523   +/-   ##
=======================================
  Coverage   83.31%   83.31%           
=======================================
  Files          27       27           
  Lines        3715     3715           
=======================================
  Hits         3095     3095           
  Misses        620      620           
```



------

[Continue to review full report at Codecov](https://codecov.io/gh/quiltdata/quilt/pull/1523?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/quiltdata/quilt/pull/1523?src=pr&el=footer). Last update [5ab5538...5ab5538](https://codecov.io/gh/quiltdata/quilt/pull/1523?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Pull request merged**